Description: Put all orphan files into the airportinspector package
 Some files were not declared as being part of a package.
Author: Julien BLACHE <jblache@debian.org>

Index: airport-utils-2/portinspector/AboutDialog.java
===================================================================
--- airport-utils-2.orig/portinspector/AboutDialog.java	2003-01-25 23:36:40.000000000 +0100
+++ airport-utils-2/portinspector/AboutDialog.java	2010-03-11 20:56:30.978759173 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 
 import java.awt.*;
Index: airport-utils-2/portinspector/AirportDiscoverer.java
===================================================================
--- airport-utils-2.orig/portinspector/AirportDiscoverer.java	2003-01-25 22:54:18.000000000 +0100
+++ airport-utils-2/portinspector/AirportDiscoverer.java	2010-03-11 20:56:31.008539416 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 
 
Index: airport-utils-2/portinspector/AirportDiscoveryInfo.java
===================================================================
--- airport-utils-2.orig/portinspector/AirportDiscoveryInfo.java	2003-01-25 22:54:40.000000000 +0100
+++ airport-utils-2/portinspector/AirportDiscoveryInfo.java	2010-03-11 20:56:31.033623779 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 import java.util.*;
 
Index: airport-utils-2/portinspector/AirportInfoRecord.java
===================================================================
--- airport-utils-2.orig/portinspector/AirportInfoRecord.java	2003-01-25 22:54:50.000000000 +0100
+++ airport-utils-2/portinspector/AirportInfoRecord.java	2010-03-11 20:56:31.058751989 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 
 import java.math.*;
Index: airport-utils-2/portinspector/ArpInfo.java
===================================================================
--- airport-utils-2.orig/portinspector/ArpInfo.java	2003-01-25 22:54:56.000000000 +0100
+++ airport-utils-2/portinspector/ArpInfo.java	2010-03-11 20:56:31.083799524 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airportinspector;
 
 
 public class ArpInfo
Index: airport-utils-2/portinspector/ArpInfoTreeMap.java
===================================================================
--- airport-utils-2.orig/portinspector/ArpInfoTreeMap.java	2003-01-25 22:55:03.000000000 +0100
+++ airport-utils-2/portinspector/ArpInfoTreeMap.java	2010-03-11 20:56:31.108882030 +0100
@@ -19,6 +19,7 @@
  *
  */
 
+package airportinspector;
 
 
 import java.util.*;
Index: airport-utils-2/portinspector/JTextAreaWriter.java
===================================================================
--- airport-utils-2.orig/portinspector/JTextAreaWriter.java	2003-01-25 22:53:37.000000000 +0100
+++ airport-utils-2/portinspector/JTextAreaWriter.java	2010-03-11 20:56:31.133971367 +0100
@@ -18,6 +18,8 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  *
  */
+
+package airportinspector;
  
  
 import java.io.*;
Index: airport-utils-2/portinspector/PortCount.java
===================================================================
--- airport-utils-2.orig/portinspector/PortCount.java	2003-01-25 22:55:10.000000000 +0100
+++ airport-utils-2/portinspector/PortCount.java	2010-03-11 20:56:31.175752116 +0100
@@ -20,6 +20,8 @@
  */
 
 
+package airportinspector;
+
 public class PortCount
 {
 
Index: airport-utils-2/portinspector/PortInfo.java
===================================================================
--- airport-utils-2.orig/portinspector/PortInfo.java	2003-01-25 22:55:17.000000000 +0100
+++ airport-utils-2/portinspector/PortInfo.java	2010-03-11 20:56:31.200806805 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airportinspector;
 
 
 public class PortInfo
Index: airport-utils-2/portinspector/PortInfoChangeListener.java
===================================================================
--- airport-utils-2.orig/portinspector/PortInfoChangeListener.java	2003-01-25 22:55:24.000000000 +0100
+++ airport-utils-2/portinspector/PortInfoChangeListener.java	2010-03-11 20:56:31.225888392 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airportinspector; 
 
 
 
Index: airport-utils-2/portinspector/PortInfoTreeMap.java
===================================================================
--- airport-utils-2.orig/portinspector/PortInfoTreeMap.java	2003-01-25 22:55:29.000000000 +0100
+++ airport-utils-2/portinspector/PortInfoTreeMap.java	2010-03-11 20:56:31.250976490 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 import java.util.*;
 import java.math.*;
Index: airport-utils-2/portinspector/PortInspector.java
===================================================================
--- airport-utils-2.orig/portinspector/PortInspector.java	2003-01-25 22:55:43.000000000 +0100
+++ airport-utils-2/portinspector/PortInspector.java	2010-03-11 20:56:31.276059514 +0100
@@ -20,7 +20,7 @@
  */
 
 
-
+package airportinspector;
 
 import java.util.*;
 import java.net.*;
Index: airport-utils-2/portinspector/PortMapTable.java
===================================================================
--- airport-utils-2.orig/portinspector/PortMapTable.java	2003-01-25 22:55:59.000000000 +0100
+++ airport-utils-2/portinspector/PortMapTable.java	2010-03-11 20:56:31.301145856 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airportinspector; 
  
 import java.awt.*;
 import java.awt.event.*;
Index: airport-utils-2/portinspector/Preferences.java
===================================================================
--- airport-utils-2.orig/portinspector/Preferences.java	2003-01-25 22:56:32.000000000 +0100
+++ airport-utils-2/portinspector/Preferences.java	2010-03-11 20:56:31.342899881 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 import java.io.*;
 import java.net.*;
Index: airport-utils-2/portinspector/PreferencesDialog.java
===================================================================
--- airport-utils-2.orig/portinspector/PreferencesDialog.java	2003-01-25 23:19:19.000000000 +0100
+++ airport-utils-2/portinspector/PreferencesDialog.java	2010-03-11 20:56:31.367977839 +0100
@@ -19,7 +19,7 @@
  *
  */
 
- 
+package airportinspector; 
  
 
 import java.awt.*;
Index: airport-utils-2/portinspector/ValueFormatException.java
===================================================================
--- airport-utils-2.orig/portinspector/ValueFormatException.java	2003-01-25 22:54:26.000000000 +0100
+++ airport-utils-2/portinspector/ValueFormatException.java	2010-03-11 20:56:31.393063024 +0100
@@ -19,7 +19,7 @@
  *
  */
 
-
+package airportinspector;
 
 
 /** 
